var/spack/repos/builtin/packages/dray/package.py

```diff
@@ -0,0 +1,332 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+import os
+import socket
+
+import llnl.util.tty as tty
+from os import environ as env
+
+
+def cmake_cache_entry(name, value, vtype=None):
+    """
+    Helper that creates CMake cache entry strings used in
+    'host-config' files.
+    """
+    if vtype is None:
+        if value == "ON" or value == "OFF":
+            vtype = "BOOL"
+        else:
+            vtype = "PATH"
+    return 'set({0} "{1}" CACHE {2} "")\n\n'.format(name, value, vtype)
+
+
+class Dray(Package, CudaPackage):
```

Comment:
This should probably be called `DevilRay` since that's what the authors call it.

Reply:
When I think back at why I did this, I have a foggy memory of pain and suffering, but exactly why has been lost to time.

Reply:
we  would like to keep the name `dray` -- the package is already in use and known as both `dray` and `DevilRay`